libraries/chain/test/name_test.cpp

```diff
@@ -0,0 +1,16 @@
+#include <boost/test/unit_test.hpp>
+
+#include "eosio/chain/name.hpp"
+
+BOOST_AUTO_TEST_SUITE(name_test)
+
+BOOST_AUTO_TEST_CASE(dafault_to_string)
```

Comment:
dafult_to_string -> default_to_string

Reply:
I don't get it

Reply:
"default" is misspelled as "dafault".  Unless, this is a trendy way of saying "the fault to string" 😄 

Either way, we should clean it up.

Reply:
oh lol